app/views/grants/_sortable_table.html.haml

```diff
@@ -22,5 +21,8 @@
         %td= grant.publish_date
         %td= grant.submission_open_date
         %td= grant.submission_close_date
-        %td= link_to 'Edit', edit_grant_path(grant) if policy(grant).edit?
-        %td= link_to 'Delete', grant, method: :delete, data: { confirm: 'Are you sure?' } if policy(grant).destroy? && grant_can_be_deleted?(grant)
+        %td
+          %ul.menu
+            %li= link_to 'Edit', edit_grant_path(grant) if policy(grant).edit?
+            %li= link_to 'Delete', grant, method: :delete, data: { confirm: 'Are you sure?' } if policy(grant).destroy? && grant_can_be_deleted?(grant)
```

Comment:
I wouldn't mind seeing delete moved further over and separated , maybe add an extra th/td?